promlog/log.go

```diff
@@ -18,12 +18,20 @@ package promlog
 
 import (
 	"os"
+	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
 )
 
+var (
+	timestampFormat = log.TimestampFormat(
+		func() time.Time { return time.Now().UTC() },
+		"2006-01-02T15:04:05.000Z07:00",
```

Comment:
You should probably add a comment as to how this is different to the usual

Reply:
Done